chromadb/api/types.py

```diff
@@ -197,7 +197,7 @@ def __call__(self: EmbeddingFunction[D], input: D) -> Embeddings:
 
         setattr(cls, "__call__", __call__)
 
-    def embed_with_retries(self, input: D, **retry_kwargs: Dict) -> Embeddings:
+    def embed_with_retries(self, input: D, **retry_kwargs: Dict[str, Any]) -> Any:
```

Comment:
would it make sense to leave the return type as `Embeddings` and for the kwargs use `Any`